include/circt/Dialect/LLHD/IR/ExtractOps.td

```diff
@@ -55,7 +55,7 @@ def LLHD_ExtsOp : LLHD_Op<"exts", [
   }];
 }
 
-def LLHD_DextsOp : LLHD_Op<"dexts", [
+def LLHD_DynamicExtractSliceOp : LLHD_Op<"dyn_extract_slice", [
```

Comment:
Seems like these should match.. either DynExtractSliceOp or dynamic_extract_slice

Reply:
I went for `DynExtractSliceOp` because I think it's an unambiguous abbreviation and avoids very long operation names. Thanks for taking the time to review!